index.js

```diff
@@ -0,0 +1,2 @@
+const nombreusuario = prompt ("Ingrese su nombre")
+alert(`Hola ${nombreusuario}!`);
```

Comment:
En este caso me parece que seria mejor que no uses interpolacion de strings.
```suggestion
alert(`Hola ` + nombreusuario);
```